.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,8 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Report a security vulnerability
-    url: https://loopback.io/doc/en/contrib/Reporting-issues.html#security-issues
-    about: Do not report security vulnerabilities using GitHub issues. Please send an email to `reachsl@us.ibm.com` instead.
```

Comment:
The goal of this section is to add a new "issue type" so that when people are about to submit a security-related bug report, we nudge them towards responsible disclosure. I would like to preserve this user interaction.

See https://github.com/strongloop/loopback-next/issues/new/choose

<img width="944" alt="Screen Shot 2020-03-05 at 14 10 22" src="https://user-images.githubusercontent.com/1140553/75984632-11f7cf00-5eeb-11ea-876b-72fe6412b1c7.png">

What is your motivation for removing this section? Can we perhaps just change the `url` field to point to a more relevant resource?

Reply:
This was to remove the repeated "Report a security vulnerability". But I agree that it probably best to preserve it to reduce the friction. I'm good with keeping/revising the link after the verdict of adopting responsible disclosure policy is settled